hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/gas/TinybarValues.java

```diff
@@ -115,17 +152,50 @@ public long childTransactionTinybarGasPrice() {
                         * childTransactionResourcePrices.congestionMultiplier());
     }
 
+    /**
+     * Returns the tinycent gas price for dispatching a child transaction based on the current exchange rate,
+     * Without denomination, but with congestion multiplier, saving the precision.
+     * @return the tinycent gas price
+     */
+    public long childTransactionTinycentGasPrice() {
+        if (childTransactionResourcePrices == null) {
+            throw new IllegalStateException("Cannot dispatch a child transaction from a query");
+        }
+        return childTransactionResourcePrices.basePrices().servicedataOrThrow().gas()
+                * childTransactionResourcePrices.congestionMultiplier();
```

Comment:
This class still confuses me.  Why is there a `/ FEE_SCHEDULE_UNITS_PER_TINYCENT` up at L151 but not here?

Reply:
It is different denomination.
Here we return the price in tynicents while at L151 we return in tinybars.